rs/protobuf/def/state/metadata/v1/metadata.proto

```diff
@@ -112,6 +113,17 @@ message SignWithThresholdContextTree {
   SignWithThresholdContext context = 2;
 }
 
+message PreSignatureStashTree {
```

Comment:
Just  question out of curiosity: Why is this named a tree? It doesn't seem to have a tree structure